lib/UserInterface/textinput/TerminalDisplayUnix.cpp

```diff
@@ -119,6 +119,7 @@ namespace textinput {
     TerminalConfigUnix::Get().TIOS()->c_lflag |= ECHOCTL|ECHOKE|ECHOE;
 #endif
     const char* TERM = getenv("TERM");
+    SetTERM(TERM);
```

Comment:
If `TERM` is `"dumb"`, the dumb flag on the `TerminalDisplay` base should be set.